libcst/codemod/_context.py

```diff
@@ -39,6 +39,12 @@ class CodemodContext:
     #: running codemods from the command line.
     filename: Optional[str] = None
 
+    #: The current package if a codemod is being executed against a file that
+    #: lives on disk, and the repository root is correctly configured. This
+    #: Will take the form of a dotted name such as ``foo.bar.baz`` for a file
+    #: in the repo named ``foo/bar/baz.py``.
+    package: Optional[str] = None
```

Comment:
I'm worrying about this name is confusing. `package` means directory with `__init__.py` file in Python.
`full_module_name` might be a better name here.

Reply:
Updated.